updated application.properties (added some properties)

vita792 opened the pull request.

vita792 committed:
updated application.properties (added some properties)

vita792 committed:
updated application.properties (added properties for auth0);
created package security with classes SecurityConfig.java, AudienceValidator.java

dimalian commented on src/main/resources/application.properties:
```diff
@@ -1,11 +1,22 @@
+spring.sql.init.mode=never
+
+#auth0
+auth0.audience=${yourApiIdentifier}
```
Rename it
```suggestion
auth0.audience=${AUTH0_AUDIENCE}
```

dimalian commented on src/main/resources/application.properties:
```diff
@@ -1,11 +1,22 @@
+
+#auth0
+auth0.audience=${yourApiIdentifier}
+spring.security.oauth2.resourceserver.jwt.issuer-uri=https://${yourDomain}/
```
```suggestion
spring.security.oauth2.resourceserver.jwt.issuer-uri=https://${OAUTH2_ISSUER_URI}/
```

vita792 committed:
updated application.properties (changed properties for auth0.audience and spring.security.oauth2.resourceserver.jwt.issuer-uri)

vita792 committed:
Merge branch 'dev' of https://github.com/store-challenge/store-api into #16_implement_auth0

 Conflicts:
	pom.xml
	src/main/resources/application.properties

vita792 committed:
updated application.properties (changed properties for auth0.audience and spring.security.oauth2.resourceserver.jwt.issuer-uri)

changed endpoints for categories to category and sub-categories to sub-category

vita792 committed:
added @CrossOrigin(origins = "*") to controllers
